tests/unit/util-test.js

```diff
@@ -182,15 +181,15 @@ describe('util', function () {
 
     it('logs timing information', function () {
       var capture = []
-      console.log = function () { capture.push(_.toArray(arguments)[0]) }
+      console.log = function () { capture.push(Array(arguments)[0])) }
       time('foo', function () {})
       expect(capture.length).to.equal(1)
       expect(capture[0]).to.match(/^foo time: .*ms/)
     })
 
     it('returns the value from the evaluated function', function () {
       console.log = function () {}
-      expect(time('foo', _.constant('bar'))).to.equal('bar')
+      expect(time('foo', 'bar')).to.equal('bar')
```

Comment:
I think the second argument might still need to be passed in as a function. https://github.com/ciena-blueplanet/dagre/blob/master/src/util.js#L209

Reply:
ah - yes, you are right. That totally passed while I developing. I don't think there is any way it should though.